src/Sulu/Bundle/CategoryBundle/DependencyInjection/SuluCategoryExtension.php

```diff
@@ -153,7 +153,7 @@ public function prepend(ContainerBuilder $container)
                                         'list_key' => 'categories',
                                         'display_properties' => ['name'],
                                         'empty_text' => 'sulu_category.no_category_selected',
-                                        'icon' => 'fa-tags',
+                                        'icon' => 'su-tag',
```

Comment:
i noticed that we use the fontawesome icon instead of the sulu icon (that is already used for the [search index](https://github.com/sulu/sulu/blob/a31dab9f85a446d1a315eb0be639c9ceaca6aa70/src/Sulu/Bundle/CategoryBundle/DependencyInjection/SuluCategoryExtension.php#L174))

Reply:
Interesting, yeah we should use in the core also our own icons. Good catch 👍